.k8s/dev/ingress.yaml

```diff
@@ -4,6 +4,12 @@ metadata:
   name: laa-court-data-ui-app-ingress
   namespace: laa-court-data-ui-dev
   annotations:
+    kubernetes.io/ingress.class: modsec01
+    nginx.ingress.kubernetes.io/enable-modsecurity: "true"
+    nginx.ingress.kubernetes.io/modsecurity-snippet: |
```

Comment:
Do we want ModSecurity ( WAF / web application firewall) ? It can cause false positives / block requests. This would mean we would need to keep an eye on it. It would need thorough testing before hand.

The apply team have been looking into it here as it blocked google oauth calls for them https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/3686039762/ModSecurity